etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiFhirImplementationTest.groovy

```diff
@@ -0,0 +1,163 @@
+package gov.hhs.cdc.trustedintermediary.external.hapi
+
+import ca.uhn.fhir.fhirpath.FhirPathExecutionException
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import org.hl7.fhir.instance.model.api.IBaseResource
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.DiagnosticReport
+import org.hl7.fhir.r4.model.ServiceRequest
+import spock.lang.Specification
+
+class HapiFhirImplementationTest extends Specification {
```

Comment:
This file is in the wrong location.  It needs to be in the `shared` subproject.

Reply:
You are welcome to add tests for `HapiFhirImplementation`, but do realize it was meant to be a "humble object" and therefore was added to the JaCoCo code coverage ignore list.  See `/build.gradle:95`.  If you keep these tests, consider removing the JaCoCo ignore statement.

Reply:
Unignoring `HapiFhirImplementation` from codecoverage because I stand by those tests!

Reply:
Nice!  I'm all for tests!